nibabel/nifti2.py

```diff
@@ -221,6 +222,17 @@ def _chk_eol_check(hdr, fix=False):
             rep.fix_msg = 'setting EOL check to 13, 10, 26, 10'
         return hdr, rep
 
+    @classmethod
+    def is_header(klass, binaryblock):
+        if len(binaryblock) < 540:
```

Comment:
I don't believe this is the correct logic; the sniff size doesn't have to be 540; the sniffed info has to indicate that the header size is 540 exactly.